.github/workflows/tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Tests
+
+on:
+  pull_request:
+    branches: [main, development, feat/*]
+  workflow_dispatch:
+
+jobs:
+  ci:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        php: ['8.1', '8.2', '8.3']
+        dependency-version: [prefer-stable]
+        parallel: ['', '--parallel']
```

Comment:
Where is this used?

Reply:
For Pest's parallel execution: https://pestphp.com/docs/optimizing-tests#content-parallel-testing

Reply:
but you are not passing this variable to the run test command, shouldn't you use `${{ matrix.parallel }}`?
https://github.com/multiversx/mx-sdk-php/actions/runs/8033201667/job/21943439564#step:6:9

Reply:
Good catch @razvanphp, the test output now shows `Parallel` info correctly as a confirmation 👍